gearmand/datadog_checks/gearmand/gearmand.py

```diff
@@ -118,3 +118,20 @@ def check(self, instance):
         except Exception as e:
             self.service_check(self.SERVICE_CHECK_NAME, AgentCheck.CRITICAL, message=str(e), tags=tags)
             raise
+
+        self._collect_metadata(client)
+
+    def _collect_metadata(self, client):
+        try:
+            resp = client.get_version()
+        except Exception as e:
+            self.log.warning('Error retrieving version information: %s', e)
+            return
+
+        if not resp.startswith('OK'):
+            self.log.warning('Error retrieving version information from server, response: %s', resp)
+            return
+
+        server_version = resp.lstrip('OK ')
```

Comment:
`[l|r]strip` removes anything in the sequence, not an exact match. maybe use `[3:]`